internal/static/identify.go

```diff
@@ -23,7 +22,15 @@ func (i *identify) PlanType() types.PlanType {
 }
 
 func (i *identify) Match(fs afero.Fs) bool {
-	return utils.HasFile(fs, "index.html", "hugo.toml")
+	hasFile := utils.HasFile(fs, "index.html", "hugo.toml")
+	if !hasFile {
+		// Determining the configuration directory mode for Hugo.
+		// https://gohugo.io/getting-started/configuration/#configuration-directory
+		hugoConfigSource := afero.NewBasePathFs(fs, "config/_default")
+		hasHugoConfigFile := utils.HasFile(hugoConfigSource, "hugo.toml")
+		return hasHugoConfigFile
+	}
+	return hasFile
```

Comment:
```suggestion
	return utils.HasFile(fs, "index.html", "hugo.toml", "config/_default/hugo.toml")
```

I did not verify if this works.

Reply:
perfect suggestion